consensus/misc/eip4844/eip4844_test.go

```diff
@@ -57,12 +57,12 @@ func TestCalcExcessBlobGas(t *testing.T) {
 	}
 	for i, tt := range tests {
 		blobGasUsed := uint64(tt.blobs) * params.BlobTxBlobGasPerBlob
-		parent := &types.Header{
+		header := &types.Header{
 			Time:          *config.CancunTime,
 			ExcessBlobGas: &tt.excess,
 			BlobGasUsed:   &blobGasUsed,
 		}
-		result := CalcExcessBlobGas(config, parent)
+		result := CalcExcessBlobGas(config, header, header)
```

Comment:
it's a bit weird that you are passing the header as its parent. I agree that it doesn't matter as only the values used in the calculation are filled in, but it will look like a bug to any random reviewer. It wouldn't hurt to add a comment so that we don't get confused ourselves, or any drive-by reviewer looking to farm airdrops. That's a nit, feel free to ignore.

Reply:
fixed up